src/OrderLinesList/OrderLinesListContainer.js

```diff
@@ -146,11 +192,55 @@ OrderLinesListContainer.manifest = Object.freeze({
     type: 'okapi',
     throwErrors: false,
   },
+  exportLines: ORDER_LINES,
   lineOrders: {
     ...ORDERS,
     accumulate: true,
     fetch: false,
   },
+  vendors: {
```

Comment:
since manifests are the same, they could be in shared HOC. Also please check that their names are unique, because somewhere could be used resources without manual fetching and they can collapse.